aces/Loris
# Update package.json / node_modules

## Brief summary of changes
1. Updates all the modules in package.json
2. Adds updated package-lock.json as a result of updated package.json
3. Updates .babelrc to refer to new @babel packages
4. Replaces UglifyJS with Terser to support ES6 (reading: https://stackoverflow.com/questions/47439067/uglifyjs-throws-unexpected-token-keyword-const-with-node-modules)

#### Testing instructions (if applicable)

1. run `make`, and make sure there are no errors in installing npm and compiling

#### Links to related tickets (GitHub, Redmine, ...)

* this PR replaces https://github.com/aces/Loris/pull/4449 for a better reviewing experience
